osf/models/preprint_provider.py

```diff
@@ -34,24 +33,34 @@ def __unicode__(self):
 
     @property
     def top_level_subjects(self):
```

Comment:
This behavior is implicitly tested by [these tests](https://github.com/mfraezz/osf.io/blob/cb114cd94c1c56eb1dbcfda24b804a490ce14c53/api_tests/preprint_providers/views/test_preprint_provider_subjects_list.py#L79-L94), but I can add a couple model tests if you want